charts/skypilot/templates/ingress.yaml

```diff
@@ -37,22 +39,11 @@ metadata:
     nginx.ingress.kubernetes.io/rewrite-target: /$2
     skypilot.co/ingress-path: {{ .Values.ingress.path }}
     {{- end }}
-    nginx.ingress.kubernetes.io/server-snippets: |
```

Comment:
This annotation was a hallucination - it doesn't do anything. Changing to `server-snippet`, which does exist, doesn't work due to duplicate `location /` blocks. Use `configuration-snippet` instead for adding stuff inside the `location /` block.